### PR TITLE
Add WriteAllBytes benchmarks

### PR DESCRIPTION
**Pull request by Jozkee**

### CODE REVIEW

**Comment by Jozkee** [comment]
cc @adamsitnik @carlossanlop

File: src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.File.cs [hunk 1]
```diff
@@ -46,5 +57,40 @@ public void Delete()
...
+            };
+        }
+
+        [Benchmark]
+        [Arguments(HalfKibibyte)]
+        [Arguments(FourKibibytes)]
+        [Arguments(SixteenKibibytes)]
+        [Arguments(OneMibibyte)]
+        [Arguments(HundredMibibytes)]
+        public void WriteAllBytes(int userBuffer) => File.WriteAllBytes(_testFilePath, _userBuffers[userBuffer]);
```
Comment (adamsitnik) [inline src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.File.cs:81]: nit: I would just  rename the arg name to `size` as it's part of benchmark ID

```suggestion
        public void WriteAllBytes(int size) => File.WriteAllBytes(_testFilePath, _userBuffers[size]);
```

File: src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.File.cs [hunk 2]
```diff
@@ -46,5 +57,40 @@ public void Delete()
...
+
+#if !NETFRAMEWORK
+        [Benchmark]
+        [Arguments(HalfKibibyte)]
+        [Arguments(FourKibibytes)]
+        [Arguments(SixteenKibibytes)]
+        [Arguments(OneMibibyte)]
+        [Arguments(HundredMibibytes)]
+        public Task WriteAllBytesAsync(int userBuffer) => File.WriteAllBytesAsync(_testFilePath, _userBuffers[userBuffer]);
```
Comment (adamsitnik) [inline src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.File.cs:90]:
```suggestion
        public Task WriteAllBytesAsync(int size) => File.WriteAllBytesAsync(_testFilePath, _userBuffers[size]);
```

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @Jozkee !

**Pull request merged**